forms/models/behaviour.js

```diff
@@ -125,7 +125,7 @@ define(function (require) {
       }
 
       // legacy calculations only expect to monitor value changes (performance)
```

Comment:
It's probably worth getting rid of this comment, too. As it no longer describes what happens now.